equinix/fabric_connection_schema.go

```diff
@@ -340,10 +341,10 @@ func createConnectionSideAccessPointRes() *schema.Resource {
 					Schema: createServiceProfileSch(),
 				},
 			},
-			"gateway": {
+			"router": {
```

Comment:
Similar to the above comment, this should be done in a way that does not break existing configurations: https://developer.hashicorp.com/terraform/plugin/sdkv2/best-practices/deprecations#renaming-a-computed-attribute

Reply:
This is now fixed with the latest commit. Please check and approve.